src/core/datastore/persisters/key-value-persister.js

```diff
@@ -0,0 +1,67 @@
+const _cache = {};
+
+export class KeyValuePersister {
+  /** @type {Boolean} */
+  _cacheEnabled;
+  /** @type {Number} */
+  _ttl;
+
+  // TODO: implement TTL, make _cache a constructor argument?
```

Comment:
Let's capture this TODO and remove `ttl` since it isn't being used.

Reply:
https://kinvey.atlassian.net/browse/MLIBZ-2236 is the related ticket in JIRA.

Reply:
The TTL is implemented, just not being used. Since the caching is toggled by a flag, I'll set its default value to false, so it will have no effect. Once we decide something, we can change that, but I think it's best to leave the code there, since it has a switch and will be set to "disabled".